Core.PoseEditor/CharaPoseController.cs

```diff
@@ -293,6 +301,11 @@ protected override void Awake()
             _siriDamROriginalRotation = _siriDamR.localRotation;
             _kosiOriginalRotation = _kosi.localRotation;
 
+            _anaOriginalRotation = _ana.localRotation;
+            _anaOriginalPosition = _ana.localPosition;
+            _anaOriginalRotationOffset = Quaternion.Inverse(_kosi.localRotation) * _ana.localRotation;
```

Comment:
The changes look good, but I can't merge this PR currently since it will crash in other games where `_ana` is null.

Reply:
Yeah, I add `_ana` which is only assigned in KK, not in other games' branches. So other game versions cannot be compiled.